src/components/Button/getHomeLinkHref.jsx

```diff
@@ -0,0 +1,6 @@
+const getHomeLinkHref = from =>
```

Comment:
- shouldn't we move `https://manager.cozycloud.cc/cozy/create` in constants?
- could you please add a simple test for this function?